# German (Germany) translations for fb_tools.
# Copyright (C) 2019 Frank Brehm, Berlin
# This file is distributed under the same license as the fb_tools project.
# Frank Brehm <frank.brehm@pixelpark.com>, 2019.
#
msgid ""
msgstr ""
"Project-Id-Version: fb_tools 1.4.2\n"
"Report-Msgid-Bugs-To: frank@brehm-online.com\n"
"POT-Creation-Date: 2019-02-12 10:20+0100\n"
"PO-Revision-Date: 2019-02-12 10:22+0100\n"
"Last-Translator: Frank Brehm <frank@brehm-online.com>\n"
"Language: de_DE\n"
"Language-Team: de_DE <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.6.0\n"

#: bin/get-file-to-remove:79
msgid ""
"{c}-Object:\n"
"{a}"
msgstr ""
"{c}-Objekt:\n"
"{a}"

#: lib/fb_tools/app.py:67
msgid "Forced execution - whatever it means."
msgstr "Forcierte Ausführung, was auch immer das bedeuten mag."

#: lib/fb_tools/app.py:146
msgid "Invalid env_prefix {!r} given - it may not be empty."
msgstr "Ungültiger env_prefix {!r} übergeben - er darf nicht leer sein."

#: lib/fb_tools/app.py:150
msgid ""
"Invalid characters found in env_prefix {!r}, only alphanumeric characters and digits and "
"underscore (this not as the first character) are allowed."
msgstr ""
"Ungültige Zeichen in env_prefix {!r} gefunden, nur alphanumerische Zeichen und Unterstriche "
"(diese aber nicht als erstes Zeichen) sind erlaubt."

#: lib/fb_tools/app.py:161
msgid "Unknown and undescriped application."
msgstr "Unbekannte und nicht beschriebene Anwendung."

#: lib/fb_tools/app.py:186
msgid "Wrong exit_value {!r}, must be >= 0."
msgstr "Falscher Wert für exit_value {!r}, muss größer als oder gleich Null sein."

#: lib/fb_tools/app.py:416
msgid "Trying to get {} via console ..."
msgstr "Versuche {} über die Konsole zu bekommen …"

#: lib/fb_tools/app.py:431
msgid "Got a signal {}."
msgstr "Erhielt ein Signal {}."

#: lib/fb_tools/app.py:434
msgid "Got a signal {n!r} ({s})."
msgstr "Erhielt ein Signal {n!r} ({s})."

#: lib/fb_tools/app.py:441
msgid "Exit on signal {n!r} ({s})."
msgstr "Beendigung auf Signal {n!r} ({s})."

#: lib/fb_tools/app.py:449
msgid "Tweaking signal handlers."
msgstr "Schraube an den Signalhandlern."

#: lib/fb_tools/app.py:455
msgid "Setting signal handler for {n!r} ({s})."
msgstr "Setze Signalhandler für {n!r} ({s})."

#: lib/fb_tools/app.py:466
msgid "Enter "
msgstr "Eingabe von "

#: lib/fb_tools/app.py:473
msgid "Repeat enter "
msgstr "Wiederholen der Eingabe von "

#: lib/fb_tools/app.py:483
msgid "{n} and repeated {n} did not match."
msgstr "{n} und das wiederholte {n} stimmen nicht überein."

#: lib/fb_tools/app.py:487
msgid "Restoring original signal handlers."
msgstr "Stelle original Signalhandler wieder her."

#: lib/fb_tools/app.py:492
msgid "Got {n!r}: {s!r}"
msgstr "Erhielt {n!r}: {s!r}"

#: lib/fb_tools/app.py:542
msgid "The application is not completely initialized."
msgstr "Die Anwendung ist nicht vollständig initialisiert."

#: lib/fb_tools/app.py:553
msgid "Object {!r} seems not to be completely initialized."
msgstr "Das {!r}-Objekt scheit noch nicht vollständig initialisiert zu sein."

#: lib/fb_tools/app.py:563
msgid "Ending."
msgstr "Beenden."

#: lib/fb_tools/app.py:582
msgid "Executing {} ..."
msgstr "Führe {} aus …"

#: lib/fb_tools/app.py:604
msgid "General options"
msgstr "Allgemeine Optionen"

#: lib/fb_tools/app.py:608
msgid "Simulation mode, nothing is really done."
msgstr "Simulations-Modus, es wird nichts wirklich verändert."

#: lib/fb_tools/app.py:619
msgid "Use colored output for messages."
msgstr "Verwenden kolorierter Ausgabe für Mitteilungen."

#: lib/fb_tools/app.py:626
msgid "Increase the verbosity level"
msgstr "Erhöhen des Ausführichkeits-Niveaus"

#: lib/fb_tools/app.py:631
msgid "Silent execution, only warnings and errors are emitted."
msgstr "Stillschweigende Ausführung, nur Warnungen und Fehler werden ausgegeben."

#: lib/fb_tools/app.py:636
msgid "Show this help message and exit."
msgstr "Zeigt diesen Hilfe-Bildschirm und beendet sich."

#: lib/fb_tools/app.py:640
msgid "Display brief usage message and exit."
msgstr "Zeigt eine kurze Darstellug zur Verwendung und beendet sich."

#: lib/fb_tools/app.py:642
#, python-format
msgid "Version of %(prog)s: {}"
msgstr "Version von %(prog)s: {}"

#: lib/fb_tools/app.py:645
msgid "Show program's version number and exit."
msgstr "Stellt die Programm-Version dar und beendet sich."

#: lib/fb_tools/app.py:779
msgid "Starting in:"
msgstr "Start in:"

#: lib/fb_tools/app.py:801
msgid "Aborted by user interrupt."
msgstr "Abbruch durch Benutzer-Unterbrechung."

#: lib/fb_tools/common.py:147
msgid "{} is a tty."
msgstr "{} ist ein tty."

#: lib/fb_tools/common.py:151
msgid "Platform is Windows and not ansi_term."
msgstr "Die Plattform ist Windows und nicht ansi_term."

#: lib/fb_tools/common.py:155
msgid "{} is not a tty."
msgstr "{} ist kein tty."

#: lib/fb_tools/common.py:485
msgid "Given value is {!r}."
msgstr "Der übergebene Wert ist {!r}."

#: lib/fb_tools/common.py:524
msgid "Could not determine bytes in {!r}."
msgstr "Konnte die Bytes nicht in {!r} ermitteln."

#: lib/fb_tools/common.py:609
msgid "Couldn't detect unit {!r}."
msgstr "Konnte die Maßeinheit in {!r} nicht ermitteln."

#: lib/fb_tools/common.py:712
msgid "Argument {a!r} must be of type {t1!r} or {t2!r}."
msgstr "Das Argument {a!r} muss vom Typ {t1!r} oder {t2!r} sein."

#: lib/fb_tools/config.py:65 lib/fb_tools/get_file_rm_app.py:388 lib/fb_tools/pdns_bulk_rm_app.py:275
msgid "File {!r} does not exists."
msgstr "Die Datei {!r} existiert nicht."

#: lib/fb_tools/config.py:68 lib/fb_tools/get_file_rm_app.py:391 lib/fb_tools/pdns_bulk_rm_app.py:279
msgid "File {!r} is not a regular file."
msgstr "Die Datei {!r} ist keine reguläre Datei."

#: lib/fb_tools/config.py:124
msgid "Encoding {v!r} must be a {s!r} object, but is a {c!r} object instead."
msgstr "Die Kodierung {v!r} muss ein {s!r}-Objekt sein, ist aber statt dessen ein {c!r}-Objekt."

#: lib/fb_tools/config.py:142
msgid "A configuration directory may not be None."
msgstr "Ein Konfigurations-Verzeichnis darf nicht None sein."

#: lib/fb_tools/config.py:158
msgid "A configuration file may not be None."
msgstr "Eine Konfigurations-Datei darf nicht None sein."

#: lib/fb_tools/config.py:163 lib/fb_tools/config.py:171
msgid "Configuration file {!r} exists, but is not a regular file."
msgstr "Die Konfigurations-Datei {!r} existiert, ist aber keine reguläre Datei."

#: lib/fb_tools/config.py:203
msgid "Searching for {!r} ..."
msgstr "Suche nach {!r} …"

#: lib/fb_tools/config.py:205
msgid "Configuration file {!r} not found."
msgstr "Die Konfigurations-Datei {!r} wurde nicht gefunden."

#: lib/fb_tools/config.py:207
msgid "Configuration file error"
msgstr "Konfigurationsdatei-Fehler"

#: lib/fb_tools/config.py:218
msgid "Reading {!r} ..."
msgstr "Lese {!r} …"

#: lib/fb_tools/config.py:229
msgid "Wrong configuration in {!r} found"
msgstr "Falsche Konfiguration in {!r} gefunden"

#: lib/fb_tools/config.py:231
msgid "Configuration parse error"
msgstr "Fehler beim Parsen der Konfiguration"

#: lib/fb_tools/config.py:254
msgid "Checking config section {!r} ..."
msgstr "Überprüfe Konfigurationsabschnitt {!r} …"

#: lib/fb_tools/errors.py:44
msgid "Wrong mail address {a!r} ({c})"
msgstr "Ungültige Mailadresse {a!r} ({c})"

#: lib/fb_tools/errors.py:107
msgid "Process with PID {pid} got signal {signal}."
msgstr "Der Prozess mit der PID {pid} hat das Signal {signal} erhalten."

#: lib/fb_tools/errors.py:146 lib/fb_tools/vsphere/errors.py:139
msgid "The network {!r} is not existing."
msgstr "Das Netzerk {!r} existiert nicht."

#: lib/fb_tools/errors.py:165 lib/fb_tools/vsphere/errors.py:158
msgid "Could not connect to the vSphere host {h}:{p} as user {u!r}."
msgstr "Konnte mich nicht zum vSphere-Host {h}:{p} als Nutzer {u!r} verbinden."

#: lib/fb_tools/errors.py:186
msgid "No SAN based datastore found with at least {m:0.0f} MiB == {g:0.1f} GiB available space."
msgstr ""
"Keinen SAN-basierten Datastore mit wenigstens {m:0.0f} MiB == {g:0.1f} GiB verfügbaren Platz "
"gefunden."

#: lib/fb_tools/errors.py:224
msgid "Method {func}() has to be overridden in class {cls!r}."
msgstr "Die Methode {func}() der Klasse {cls!r} muss überschrieben werden."

#: lib/fb_tools/errors.py:256
msgid " (timeout after {:0.1f} secs)"
msgstr " (Timeout nach {:0.1f} Sekunden)"

#: lib/fb_tools/errors.py:283
msgid "Timeout error on reading"
msgstr "Timeout-Fehler beim Lesen"

#: lib/fb_tools/errors.py:305
msgid "Timeout error on writing"
msgstr "Timeout-Fehler beim Schreiben"

#: lib/fb_tools/errors.py:328
msgid "Unknown OS command."
msgstr "Unbekanntes Betriebssystem-Kommando."

#: lib/fb_tools/errors.py:335
msgid "Empty command list given."
msgstr "Leere Kommando-Liste übergeben."

#: lib/fb_tools/errors.py:346
#, fuzzy
msgid "Could not found OS command:"
msgid_plural "Could not found OS commands:"
msgstr[0] "Konnte das Betriebssystem-Kommando nicht finden:"
msgstr[1] "Konnte die Betriebssystem-Kommandos nicht finden:"

#: lib/fb_tools/errors.py:379
msgid "Couldn't occupy lockfile {lf!r} in {d:0.1f} seconds with {tries} tries."
msgstr "Konnte die Lock-Datei {lf!r} nicht in {d:0.1f} Sekunden mit {tries} Versuchen vereinnahmen."

#: lib/fb_tools/get_file_rm_app.py:61
msgid "Value must be at least {m} - {v} was given."
msgstr "Der Wert muss mindestens {m} sein - {v} wurde gegeben."

#: lib/fb_tools/get_file_rm_app.py:103
msgid "The given pattern {!r} is not a valid date pattern"
msgstr "Das übergebene Muster {!r} is kein gültiges Datums-Muster"

#: lib/fb_tools/get_file_rm_app.py:107
#, python-format
msgid ". The must be exactly one occurence of '%Y', one of '%m' and one of '%d'."
msgstr ". Es muss exakt einmal '%Y', einmal '%m' und einmal '%d' angegeben werden."

#: lib/fb_tools/get_file_rm_app.py:136
msgid ""
"Returns a newline separated list of files generated from file globbing patterns given as "
"arguments to this application, where all files are omitted, which should not be removed."
msgstr ""
"Gibt eine mit Zeilenumbruch separierte Liste von Dateien zurück, die aus dem Datei-Muster, "
"welches als Kommando-Zeilen-Argument dieser Anwendung übergeben wurde, generiert wurde, wobei "
"alle Datein fehlen, welche nicht gelöscht werden sollen."

#: lib/fb_tools/get_file_rm_app.py:175 lib/fb_tools/get_file_rm_app.py:191
#: lib/fb_tools/get_file_rm_app.py:207 lib/fb_tools/get_file_rm_app.py:223
#: lib/fb_tools/get_file_rm_app.py:239
msgid "Wrong value {v!r} for {n}, must be >= {m}"
msgstr "Ungültiger Wert {v!r} für {n}, muss größer als oder gleich {m} sein"

#: lib/fb_tools/get_file_rm_app.py:264
msgid "Keep options"
msgstr "Optionen zum Aufbewahren"

#: lib/fb_tools/get_file_rm_app.py:267
msgid "NR_FILES"
msgstr "ANZAHL_DATEIEN"

#: lib/fb_tools/get_file_rm_app.py:269
msgid "How many of the last files should be kept (default: {default}, minimum: {min})?"
msgstr "Wieviele der letzten Dateien sollen aufbewahrt werden (Vorgabe: {default}, Minimum {min})?"

#: lib/fb_tools/get_file_rm_app.py:276
msgid "DAYS"
msgstr "TAGE"

#: lib/fb_tools/get_file_rm_app.py:278
msgid "How many files one per day from today on should be kept (default: {default}, minimum: {min})?"
msgstr ""
"Wieviele Dateien (eine pro Tag) sollen von heute an rückwärts aufbewahrt werden (Vorgabe:  "
"{default}, Minimum {min})?"

#: lib/fb_tools/get_file_rm_app.py:285
msgid "WEEKS"
msgstr "WOCHEN"

#: lib/fb_tools/get_file_rm_app.py:287
msgid "How many files one per week from today on should be kept (default: {default}, minimum: {min})?"
msgstr ""
"Wieviele Dateien (eine pro Woche) sollen von heute an rückwärts aufbewahrt werden (Vorgabe:  "
"{default}, Minimum {min})?"

#: lib/fb_tools/get_file_rm_app.py:294
msgid "MONTHS"
msgstr "MONATE"

#: lib/fb_tools/get_file_rm_app.py:296
msgid "How many files one per month from today on should be kept (default: {default}, minimum: {min})?"
msgstr ""
"Wieviele Dateien (eine pro Monat) sollen von heute an rückwärts aufbewahrt werden (Vorgabe:  "
"{default}, Minimum {min})?"

#: lib/fb_tools/get_file_rm_app.py:303
msgid "YEARS"
msgstr "JAHRE"

#: lib/fb_tools/get_file_rm_app.py:305
msgid "How many files one per year from today on should be kept (default: {default}, minimum: {min})?"
msgstr ""
"Wieviele Dateien (eine pro Jahr) sollen von heute an rückwärts aufbewahrt werden (Vorgabe:  "
"{default}, Minimum {min})?"

#: lib/fb_tools/get_file_rm_app.py:312 lib/fb_tools/get_vm_app.py:157
#: lib/fb_tools/get_vm_list_app.py:161 lib/fb_tools/pdns_bulk_rm_app.py:189
#: lib/fb_tools/pdns_bulk_rm_app.py:236
msgid "FILE"
msgstr "DATEI"

#: lib/fb_tools/get_file_rm_app.py:313
msgid "File pattern to generate list of files to remove."
msgstr "Dateimuster, um die Liste der zu löschenden Dateien zu generieren."

#: lib/fb_tools/get_file_rm_app.py:341
msgid "Resolving date pattern {!r}."
msgstr "Löse Datumsmuster {!r} auf."

#: lib/fb_tools/get_file_rm_app.py:365
msgid "Checking given files..."
msgstr "Überprüfe übergebene Dateien …"

#: lib/fb_tools/get_file_rm_app.py:370
msgid "Checking given file {!r} ..."
msgstr "Überprüfe übergebene Datei {!r} …"

#: lib/fb_tools/get_file_rm_app.py:379
msgid "Resolved paths:"
msgstr "Aufgelöste Pfade:"

#: lib/fb_tools/get_file_rm_app.py:381
msgid "File pattern {!r} does not match any files."
msgstr "Das Dateimuster {!r} passt auf keine Dateien."

#: lib/fb_tools/get_file_rm_app.py:386
msgid "Checking {!r} ..."
msgstr "Überprüfe {!r} …"

#: lib/fb_tools/get_file_rm_app.py:396
msgid "File {fi!r} does not match pattern {pa!r}."
msgstr "Die Datei {fi!r} passt nicht in das Dateimuster {pa!r}."

#: lib/fb_tools/get_file_rm_app.py:406
msgid "Date in file {fi!r} is not a valid date: {e}."
msgstr "Das Datum in der Datei {fi!r} ist kein gültiges Datum: {e}."

#: lib/fb_tools/get_file_rm_app.py:458
msgid "Did not found any files to evaluate."
msgstr "Fand keine Dateien zur Auswertung"

#: lib/fb_tools/get_file_rm_app.py:491
msgid "Keeping last file ..."
msgid_plural "Keeping last {} files ..."
msgstr[0] "Behalte die letzte Datei …"
msgstr[1] "Behalte die letzten {} Dateien …"

#: lib/fb_tools/get_file_rm_app.py:497
msgid "Keep last file {!r}."
msgstr "Behalte die letzte Datei {!r}."

#: lib/fb_tools/get_file_rm_app.py:502
msgid "Files to keep:"
msgstr "Zu behaltende Dateien:"

#: lib/fb_tools/get_file_rm_app.py:523
msgid "Files to keep for year:"
msgstr "Dateien, die für Jahr zu behalten sind:"

#: lib/fb_tools/get_file_rm_app.py:545
msgid "Got last month: {!r}"
msgstr "Ermittelter letzter Monat: {!r}"

#: lib/fb_tools/get_file_rm_app.py:559
msgid "Files to keep for month:"
msgstr "Dateien, die für Monat zu behalten sind:"

#: lib/fb_tools/get_file_rm_app.py:573
msgid "Got last Monday: {!r}"
msgstr "Ermittelter letzter Montag: {!r}"

#: lib/fb_tools/get_file_rm_app.py:586
msgid "Files to keep for week:"
msgstr "Dateien, die für Woche zu behalten sind:"

#: lib/fb_tools/get_file_rm_app.py:599
msgid "Got last day: {!r}"
msgstr "Ermittelter letzter Tag: {!r}"

#: lib/fb_tools/get_file_rm_app.py:610
msgid "Keeping all files from today."
msgstr "Aufheben aller Dateien von heute."

#: lib/fb_tools/get_file_rm_app.py:617
msgid "Files to keep for day:"
msgstr "Dateien, die für Tag zu behalten sind:"

#: lib/fb_tools/get_file_rm_app.py:634
msgid "Trying to get date of file {!r}."
msgstr "Versuche das Datum aus der Datei {!r} zu entnehmen."

#: lib/fb_tools/get_file_rm_app.py:673
msgid "Explored and assigned files:"
msgstr "Ermittelte und zugewiesene Dateien:"

#: lib/fb_tools/get_vm_app.py:58
msgid "Tries to get information about the given virtual machines in VMWare VSphere and print it out."
msgstr ""
"Versucht, Informationen über Virtuelle Maschinen in VMWare VSphere zu ermitteln und gibt diese "
"aus."

#: lib/fb_tools/get_vm_app.py:132 lib/fb_tools/get_vm_list_app.py:136
msgid "Enter password for host {h!r} and user {u!r}: "
msgstr "Eingabe des Passwortes für Host {h!r} und Benutzer {u!r}: "

#: lib/fb_tools/get_vm_app.py:159 lib/fb_tools/get_vm_list_app.py:163
#: lib/fb_tools/pdns_bulk_rm_app.py:191
msgid "Configuration file (default: {!r})"
msgstr "Konfigurationsdatei (Vorgabe: {!r})"

#: lib/fb_tools/get_vm_app.py:162 lib/fb_tools/get_vm_list_app.py:166
msgid "VMWare options"
msgstr "VMWare-Optionen"

#: lib/fb_tools/get_vm_app.py:166 lib/fb_tools/get_vm_list_app.py:170
msgid "Remote vSphere host to connect to (Default: {!r})."
msgstr "Der vSphere-Server, zu dem sich verbunden werden soll (Vorgabe: {!r})"

#: lib/fb_tools/get_vm_app.py:172 lib/fb_tools/get_vm_list_app.py:176
msgid "Port on vSphere host to connect on (Default: {})."
msgstr "Der Port des vSphere-Servers, zu dem sich verbunden werden soll (Vorgabe: {!r})"

#: lib/fb_tools/get_vm_app.py:177 lib/fb_tools/get_vm_list_app.py:181
msgid "USER"
msgstr "NUTZER"

#: lib/fb_tools/get_vm_app.py:178 lib/fb_tools/get_vm_list_app.py:182
msgid "User name to use when connecting to vSphere host (Default: {!r})."
msgstr "Der Nutzername, der zum Verbinden zum vSphere-Server verwendet werden soll (Vorgabe: {!r})"

#: lib/fb_tools/get_vm_app.py:183 lib/fb_tools/get_vm_list_app.py:187
msgid "PASSWORD"
msgstr "PASSWORT"

#: lib/fb_tools/get_vm_app.py:184 lib/fb_tools/get_vm_list_app.py:188
msgid "Password to use when connecting to vSphere host."
msgstr "Passwort, das zum Verbinden zum vSphere-Server verwendet werden soll."

#: lib/fb_tools/get_vm_app.py:189
msgid "Names of the VM to get information."
msgstr "Die Namen der VMs, über die Informationen ermittelt werden sollen."

#: lib/fb_tools/get_vm_app.py:220 lib/fb_tools/get_vm_list_app.py:216
msgid "Starting {a!r}, version {v!r} ..."
msgstr "Starte {a!r}, Version {v!r} …"

#: lib/fb_tools/get_vm_app.py:228 lib/fb_tools/get_vm_list_app.py:224
msgid "Closing ..."
msgstr "Schließe …"

#: lib/fb_tools/get_vm_app.py:245
msgid "NOT FOUND"
msgstr "NICHT GEFUNDEN"

#: lib/fb_tools/get_vm_list_app.py:59
msgid "Tries to get a list of all virtual machines in VMWare VSphere and print it out."
msgstr "Versuche eine Liste aller Virtueller Maschinen in VMWare VSphere zu ermitteln und giebe sie aus."

#: lib/fb_tools/handler/__init__.py:153
msgid "Invalid time zone name {!r}."
msgstr "Ungültiger Zeitzonenname {!r}."

#: lib/fb_tools/handler/__init__.py:155
msgid "Setting time zone to {!r}."
msgstr "Setze Zeitzone auf {!r}."

#: lib/fb_tools/handler/__init__.py:159
msgid "Name of the time zone: {!r}."
msgstr "Name der Zeitzone: {!r}."

#: lib/fb_tools/handler/__init__.py:166
msgid "{}-object not initialized."
msgstr "Das {}-Objekt ist noch nicht initialisiert."

#: lib/fb_tools/handler/__init__.py:168
msgid "Method {} must be overridden in descendant classes."
msgstr "Die Methode {} muss in abgeleiteten Klassen überschrieben werden."

#: lib/fb_tools/handler/__init__.py:246
msgid "Quiet execution."
msgstr "Stillschweigende Ausführung."

#: lib/fb_tools/handler/__init__.py:294
msgid "Starting synchronous communication with '{}'."
msgstr "Starte synchrone Kommunikation mit '{}'."

#: lib/fb_tools/handler/__init__.py:298
msgid "Finished communication with '{}'."
msgstr "Kommunikation mit '{}' beendet."

#: lib/fb_tools/handler/__init__.py:312
msgid "Parameter {p!r} is not of type {t!r}."
msgstr "Der Parameter {p!r} ist nicht vom Typ {t!r}."

#: lib/fb_tools/handler/__init__.py:317
msgid "Got completed process:"
msgstr "Vollendeten Prozess erhalten:"

#: lib/fb_tools/handler/__init__.py:321 lib/fb_tools/handler/__init__.py:332
msgid "Output on {}:"
msgstr "Ausgabe an {}:"

#: lib/fb_tools/handler/__init__.py:353
msgid "Starting asynchronous communication with '{cmd}', heartbeat interval is {interval:0.1f} seconds."
msgstr "Starte asynchrone Kommunikation mit '{cmd}', Herzschlag-Intervall ist {interval:0.1f} Sekunden."

#: lib/fb_tools/handler/__init__.py:368
msgid "Checking for the end of the communication ..."
msgstr "Überprüfe das Ende der Kummunikation …"

#: lib/fb_tools/handler/__init__.py:378
msgid "Time to execute the heartbeat handler."
msgstr "Es ist an der Zeit, den Herzschlag-Handler auszuführen."

#: lib/fb_tools/handler/__init__.py:382
msgid "Sleeping {:0.2f} seconds ..."
msgstr "Schlafe für {:0.2f} Sekunden …"

#: lib/fb_tools/handler/__init__.py:390 lib/fb_tools/handler/__init__.py:399
msgid "   {w} is now: {o!r}"
msgstr "   {w} ist jetzt: {o!r}"

#: lib/fb_tools/handler/lock.py:94
msgid "Locking directory {!r} doesn't exists or is not a directory."
msgstr "Das Lockverzeichnis {!r} existiert entweder nicht oder ist kein Verzeichnis."

#: lib/fb_tools/handler/lock.py:121 lib/fb_tools/handler/lock.py:805
msgid "Locking directory {!r} isn't writeable."
msgstr "Kein Schreibzugriff auf das Lockverzeichnis {!r}."

#: lib/fb_tools/handler/lock.py:182
msgid "No lockfile given on init of a LockObject object."
msgstr "Keine Lockdatei bei der Initialisierung des LockObject-Ojektes angegeben."

#: lib/fb_tools/handler/lock.py:187 lib/fb_tools/handler/lock.py:1046
msgid "Lockfile {!r} doesn't exists."
msgstr "Die Lockdatei {!r} existiert nicht."

#: lib/fb_tools/handler/lock.py:190
msgid "Lockfile {!r} is not a regular file."
msgstr "Die Lockdatei {!r} ist keine reguläre Datei."

#: lib/fb_tools/handler/lock.py:343
msgid "Closing file descriptor {} ..."
msgstr "Schließe Datei-Deskriptor {} …"

#: lib/fb_tools/handler/lock.py:354
msgid "Automatic removing of {!r} ..."
msgstr "Automatisches Löschen von {!r} …"

#: lib/fb_tools/handler/lock.py:386
msgid "Refreshing atime and mtime of {!r} to the current timestamp."
msgstr "Setze atime und mtime von {!r} auf den aktuellen Zeitstempel."

#: lib/fb_tools/handler/lock.py:523 lib/fb_tools/handler/lock.py:545 lib/fb_tools/handler/lock.py:567
#: lib/fb_tools/handler/lock.py:590 lib/fb_tools/handler/lock.py:689 lib/fb_tools/handler/lock.py:1035
msgid "Value {val!r} for {what} is not a Number."
msgstr "Der Wert {val!r} für {what} ist keine Zahl."

#: lib/fb_tools/handler/lock.py:528 lib/fb_tools/handler/lock.py:550 lib/fb_tools/handler/lock.py:572
#: lib/fb_tools/handler/lock.py:595 lib/fb_tools/handler/lock.py:694 lib/fb_tools/handler/lock.py:1039
msgid "The value for {what} must be greater than zero (is {val!r})."
msgstr "Der Wert für {what} muss größer als Null sein (ist {val!r})."

#: lib/fb_tools/handler/lock.py:699
msgid "The value for {what} must be greater than or equal to zero (is {val!r})."
msgstr "Der Wert für {what} muss größer als oder gleich Null sein (ist {val!r})."

#: lib/fb_tools/handler/lock.py:785
msgid "Invalid PID {} given on calling create_lockfile()."
msgstr "Ungültige PID {} beim Aufruf von create_lockfile() übergeben."

#: lib/fb_tools/handler/lock.py:794
msgid "Using lock directory {!r} ..."
msgstr "Verwende Lock-Verzeichnis {!r} …"

#: lib/fb_tools/handler/lock.py:802
msgid "Trying to lock lockfile {!r} ..."
msgstr "Versuche Lockdatei {!r} zu okkupieren …"

#: lib/fb_tools/handler/lock.py:843
msgid "Current time difference: {:0.3f} seconds."
msgstr "Aktuelle Zeitdifferenz: {:0.3f} Sekunden."

#: lib/fb_tools/handler/lock.py:848
msgid "Try {try_nr} on creating lockfile {lfile!r} ..."
msgstr "Versuch Nummer {try_nr} zum Erstellen der Lockdatei {lfile!r} …"

#: lib/fb_tools/handler/lock.py:859 lib/fb_tools/handler/lock.py:980
msgid "Removing lockfile {!r} ..."
msgstr "Lösche Lockdatei {!r} …"

#: lib/fb_tools/handler/lock.py:864 lib/fb_tools/handler/lock.py:988
msgid "Error on removing lockfile {lfile!r}: {err}"
msgstr "Fehler beim Löschen der Lockdatei {lfile!r): {err}"

#: lib/fb_tools/handler/lock.py:877
msgid "Sleeping for {:0.1f} seconds."
msgstr "Schlafe für {:0.1f} Sekunden."

#: lib/fb_tools/handler/lock.py:892
msgid "Got a lock for lockfile {!r}."
msgstr "Habe einen Lock für die Lockdatei {!r} erhalten."

#: lib/fb_tools/handler/lock.py:898
msgid "Write {what!r} in lockfile {lfile!r} ..."
msgstr "Schreibe {what!r} in Lockdatei {lfile!r} …"

#: lib/fb_tools/handler/lock.py:907
msgid "Seeking and syncing {!r} ..."
msgstr "Seeken und synchronisieren von {!r} …"

#: lib/fb_tools/handler/lock.py:937
msgid "Trying to open {!r} exclusive ..."
msgstr "Versuche {!r} exklusiv zu öffnen …"

#: lib/fb_tools/handler/lock.py:939
msgid "Simulation mode, no real creation of a lockfile."
msgstr "Simulationsmodus, Lockdatei wird in Wirklichkeit nicht erstellt."

#: lib/fb_tools/handler/lock.py:947
msgid "Error on creating lockfile {lfile!r}: {err}"
msgstr "Fehler beim Erstellen der Lockdatei {lfile!r}: {err}"

#: lib/fb_tools/handler/lock.py:977
msgid "Lockfile {!r} to remove doesn't exists."
msgstr "Die zu löschende Lockdatei {!r} existiert nicht."

#: lib/fb_tools/handler/lock.py:982
msgid "Simulation mode - lockfile won't removed."
msgstr "Simulationsmodus - die Lockdatei wird nicht gelöscht."

#: lib/fb_tools/handler/lock.py:1043
msgid "Checking lockfile {!r} ..."
msgstr "Überprüfe Lockdatei {!r} …"

#: lib/fb_tools/handler/lock.py:1050
msgid "No read access for lockfile {!r}."
msgstr "Kein Lesezugriff auf Lockdatei {!r}."

#: lib/fb_tools/handler/lock.py:1054
msgid "No write access for lockfile {!r}."
msgstr "Kein Schreibzugriff auf Lockdatei {!r}."

#: lib/fb_tools/handler/lock.py:1060
msgid "Unusable lockfile {!r}."
msgstr "Nicht verwendungsfähige Lockdatei {!r}."

#: lib/fb_tools/handler/lock.py:1063
msgid "Process with PID {} is unfortunately dead."
msgstr "Der Prozess mit der PID {} ist unglücklicherweise verstorben."

#: lib/fb_tools/handler/lock.py:1066
msgid "Process with PID {} is still running."
msgstr "Der Prozess mit der PID {} läuft aktuell noch."

#: lib/fb_tools/handler/lock.py:1074
msgid "Could not stat for file {lfile!r}: {err}"
msgstr "Konnte Stat-Daten für Datei {lfile!r} nicht ermitteln: {err}"

#: lib/fb_tools/handler/lock.py:1081
msgid "Lockfile {lfile!r} is older than {max} seconds ({age} seconds)."
msgstr "Die Lockdatei {lfile!r} ist älter als {max} Sekunden ({age} Sekunden)."

#: lib/fb_tools/handler/lock.py:1084
msgid "Lockfile {lfile!r} is {age} seconds old, but not old enough ({max} seconds)."
msgstr "Die Lockdatei {lfile!r} ist {age} Sekunden alt, aber nicht alte genug ({max} Sekunden)."

#: lib/fb_tools/handler/lock.py:1112
msgid "Trying to open pidfile {!r} ..."
msgstr "Versuche PID-Datei {!r} zu öffnen …"

#: lib/fb_tools/handler/lock.py:1116
msgid "Could not open pidfile {!r} for reading:"
msgstr "Konnte PID-Datei {!r} nicht zum Lesen öffnen:"

#: lib/fb_tools/handler/lock.py:1129
msgid "First line of pidfile {!r} was empty."
msgstr "Die erste Zeile der PID-Datei {!r} war leer."

#: lib/fb_tools/handler/lock.py:1140
msgid "Could not interprete {cont!r} as a PID from {file!r}: {err}"
msgstr "Konnte {cont!r} nicht als PID (in {file!r}) interpretieren: {err}"

#: lib/fb_tools/handler/lock.py:1149
msgid "Invalid PID {pid} in {file!r} found."
msgstr "Ungültige PID {pid} in Datei {file!r} gefunden."

#: lib/fb_tools/handling_obj.py:63
msgid "Timeout on communicating with process."
msgstr "Zeitüberschreitung in der Kommunikation mit einem Prozess."

#: lib/fb_tools/handling_obj.py:88
msgid "Command {c!r} returned non-zero exit status {rc}."
msgstr "Das Kommando {c!r} gab den Nicht-Null Wert {rc} zurück."

#: lib/fb_tools/handling_obj.py:125
msgid "Command {c!r} timed out after {s} second."
msgid_plural "Command {c!r} timed out after {s} seconds."
msgstr[0] "Das Kommando {c!r} timte nach einer Sekunde aus."
msgstr[1] "Das Kommando {c!r} timte nach {s} Sekunden aus."

#: lib/fb_tools/handling_obj.py:286
msgid "Searching for command {!r} ..."
msgstr "Suche nach Befehl {!r} …"

#: lib/fb_tools/handling_obj.py:291
msgid "Command {!r} doesn't exists."
msgstr "Das Befehl {!r} existiert nicht."

#: lib/fb_tools/handling_obj.py:294 lib/fb_tools/handling_obj.py:318
msgid "Command {!r} is not executable."
msgstr "Das Befehl {!r} ist nicht ausführbar."

#: lib/fb_tools/handling_obj.py:308
msgid "Searching command in {!r} ..."
msgstr "Suche Befehl in {!r} …"

#: lib/fb_tools/handling_obj.py:321
msgid "Command {!r} not found."
msgstr "Das Kommando {!r} wurde nicht gefunden."

#: lib/fb_tools/handling_obj.py:394
msgid "STDIN and input arguments may not both be used."
msgstr "STDIN und Eingabe-Argumente dürfen nicht zusammen verwendet werden."

#: lib/fb_tools/handling_obj.py:397
msgid "Executing command args:"
msgstr "Auszuführende Befehlsargumente:"

#: lib/fb_tools/handling_obj.py:400
msgid "Performing argument {!r}."
msgstr "Verarbeite Argument {!r}."

#: lib/fb_tools/handler/__init__.py:243 lib/fb_tools/handling_obj.py:405
msgid "Executing: {}"
msgstr "Ausführung: {}"

#: lib/fb_tools/handling_obj.py:408
msgid "Simulation mode, not executing: {}"
msgstr "Simulations-Modus, führe nicht aus: {}"

#: lib/fb_tools/handling_obj.py:416
msgid "PID of process: {}"
msgstr "PID des Prozesses: {}"

#: lib/fb_tools/handling_obj.py:422
msgid "{c} happened, killing process: {e}"
msgstr "{} aufgetreten, töte Prozess: {e}"

#: lib/fb_tools/handling_obj.py:522
msgid "Nothing to do on signal."
msgstr "Aud das Signal gibt es nichts zu tun."

#: lib/fb_tools/handling_obj.py:576 lib/fb_tools/handling_obj.py:664
msgid "File doesn't exists."
msgstr "Die Datei existiert nicht."

#: lib/fb_tools/handling_obj.py:579
msgid "Read permission denied."
msgstr "Lesezugriff verboten."

#: lib/fb_tools/handling_obj.py:582
msgid "Reading file content of {!r} ..."
msgstr "Lese Inhalt der Datei {!r} …"

#: lib/fb_tools/handling_obj.py:669 lib/fb_tools/handling_obj.py:676
msgid "Write permission to {!r} denied."
msgstr "Schreibzugriff auf {!r} verboten."

#: lib/fb_tools/handling_obj.py:671 lib/fb_tools/handling_obj.py:678
msgid "Write permission denied."
msgstr "Schreibzugriff verboten."

#: lib/fb_tools/handling_obj.py:682
msgid "Write {what!r} into {to!r}."
msgstr "Schreibe {what!r} nach {to!r}."

#: lib/fb_tools/handling_obj.py:684
msgid "Writing {!r} ..."
msgstr "Schreibe {!r} …"

#: lib/fb_tools/handling_obj.py:696
msgid "Simulating write into {!r}."
msgstr "Simuliere das Schreiben nach {!r}."

#: lib/fb_tools/handling_obj.py:704
msgid "Opening {!r} for write unbuffered ..."
msgstr "Öffne {!r} zum ungepufferten Schreiben …"

#: lib/fb_tools/handler/lock.py:911 lib/fb_tools/handling_obj.py:708
msgid "Closing {!r} ..."
msgstr "Schließe {!r} …"

#: lib/fb_tools/handling_obj.py:749 lib/fb_tools/handling_obj.py:755
#: lib/fb_tools/vsphere/cluster.py:261 lib/fb_tools/vsphere/dc.py:135 lib/fb_tools/vsphere/ds.py:202
#: lib/fb_tools/vsphere/ds_cluster.py:106 lib/fb_tools/vsphere/iface.py:58
#: lib/fb_tools/vsphere/network.py:120 lib/fb_tools/vsphere/server.py:527
msgid "Parameter {t!r} must be a {e}, {v!r} was given."
msgstr "Der Parameter {t!r} muss ein {e} ein, statt dessen wurde {v!r} übergeben."

#: lib/fb_tools/handling_obj.py:816
msgid "Completed process"
msgstr "Fertiggestellter Prozess"

#: lib/fb_tools/obj.py:55
msgid "The base directory {!r} is not existing or not a directory."
msgstr "Das Basis-Verzeichnis {!r} existiert nicht oder ist kein Verzeichnis."

#: lib/fb_tools/obj.py:101 lib/fb_tools/obj.py:157
msgid "Wrong verbose level {!r}, must be >= 0"
msgstr "Falsches Ausführlichkeits-Niveau, muss größer als oder gleich Null sein"

#: lib/fb_tools/obj.py:181
msgid "Base directory {!r} does not exists."
msgstr "Das Basis-Verzeichnis {!r} existiert nicht."

#: lib/fb_tools/obj.py:184
msgid "Path for base directory {!r} is not a directory."
msgstr "Der Pfad zum Basis-Verzeichnis {!r} is kein Verzeichnis"

#: lib/fb_tools/obj.py:267
msgid "undefined error."
msgstr "unbestimmter Fehler."

#: lib/fb_tools/obj.py:276
msgid "Exception happened"
msgstr "Aufgetretene Ausnahme"

#: lib/fb_tools/pdns/__init__.py:99
msgid "Setting Loglevel of the {m} module to {ll}."
msgstr "Setze Loglevel des {m}-Moduls auf {ll}."

#: lib/fb_tools/pdns/__init__.py:137
msgid "Invalid port number {!r} for the PowerDNS API, must be 0 < PORT < 65536."
msgstr "Ungültige Portnummer {!r} für die PowerDNS-API, muss 0 < PORT < 65536 sein."

#: lib/fb_tools/pdns/__init__.py:190
msgid "The path prefix {!r} must be an absolute path."
msgstr "Der Pfad-Prefix muss ein absoluter Pfad sein."

#: lib/fb_tools/pdns/__init__.py:206
msgid "Invalid timeout {!r} for requesting the PowerDNS API, must be 0 < SECONDS < 3600."
msgstr "Ungültiger Timeout zur Abfrage der PowerDNS-API, muss 0 < SEKUNDEN < 3600 sein."

#: lib/fb_tools/pdns/__init__.py:223
msgid "Invalid user agent {!r} given."
msgstr "Ungültiger User-Agent {!r} übergeben."

#: lib/fb_tools/pdns/__init__.py:235
msgid "Invalid API server name {!r} given."
msgstr "Ungültiger API-Servername {!r} übergeben."

#: lib/fb_tools/pdns/__init__.py:282
msgid "Failed to extract request ID from response header 'location': {!r}"
msgstr "Konnte Request-ID nicht aus dem Rückgabe-Header 'location' ermitteln: {!r}"

#: lib/fb_tools/pdns/__init__.py:290
msgid "The path {!r} must be an absolute path."
msgstr "Der Pfad muss ein absoluter Pfad sein."

#: lib/fb_tools/pdns/__init__.py:308
msgid "Used URL: {!r}"
msgstr "Verwendete URL: {!r}"

#: lib/fb_tools/pdns/__init__.py:324
msgid "Request method: {!r}"
msgstr "Request-Methode: {!r}"

#: lib/fb_tools/pdns/__init__.py:346
msgid "Simulation mode, Request will not be sent."
msgstr "Simulationsmodus, der Request wird nicht gesendet."

#: lib/fb_tools/pdns/__init__.py:359 lib/fb_tools/vsphere/server.py:158
msgid "Got a {c} on connecting to {h!r}: {e}."
msgstr "Bekam ein {c} bei der Verbindung zu {h!r}: {e}."

#: lib/fb_tools/pdns/__init__.py:367
msgid "Failed to parse the response"
msgstr "Konnte Rückgabe nicht auswerten"

#: lib/fb_tools/pdns/__init__.py:392
msgid "Got an error response code {code}: {msg}"
msgstr "Erhaltener Fehler-Antortcode {code}: {msg}"

#: lib/fb_tools/pdns/__init__.py:422
msgid "Name {!r} is not a valid IP address."
msgstr "Der Name {!r} ist keine gültige IP-Adresse."

#: lib/fb_tools/pdns/__init__.py:427
msgid "Invalid FQDN {!r}."
msgstr "Ungültiger FQDN {!r}."

#: lib/fb_tools/pdns/errors.py:52
msgid "Could not interprete SOA data: {!r}."
msgstr "Konnte SOA-Daten nicht interpretieren: {!r}."

#: lib/fb_tools/pdns/record.py:55 lib/fb_tools/pdns/record.py:59
msgid "Argument {a} {v!r} must be a {o} object."
msgstr "Das Argument {a} {v!r} muss ein {o}-Objekt sein."

#: lib/fb_tools/pdns/record.py:171 lib/fb_tools/pdns/record.py:380 lib/fb_tools/pdns/record.py:781
#: lib/fb_tools/vsphere/cluster.py:246 lib/fb_tools/vsphere/dc.py:120 lib/fb_tools/vsphere/ds.py:320
#: lib/fb_tools/vsphere/ds_cluster.py:162 lib/fb_tools/vsphere/network.py:183
msgid "Comparing {} objects ..."
msgstr "Vergleiche {}-Objekte …"

#: lib/fb_tools/pdns/record.py:368
msgid "Copying current {}-object in a new one."
msgstr "Kopiere das aktuelle {}-Objekt in ein neues."

#: lib/fb_tools/pdns/record.py:423
msgid "Serial overflow - old serial {o} is in future, new serial {n} has reached its maximum value."
msgstr ""
"Überlauf der Seriennummer - die alte Seriennummer liegt in der Zukunft, die neue Seriennummer "
"{n} hat seinen Maimalwert erreicht."

#: lib/fb_tools/pdns/record.py:439 lib/fb_tools/pdns/record.py:821
msgid "Invalid type {t!r} as an item of a {c}, only {o} objects are allowed."
msgstr "Ungültiger Typ {t!r} als ein Eintrag von {c}, nor {o}-Objekte sind erlaubt."

#: lib/fb_tools/pdns/record.py:457 lib/fb_tools/pdns/record.py:839
msgid "{m} takes at most {max} arguments ({n} given)."
msgstr "{m} akzeptiert höchstens {max} Argumente ({n} wurden gegeben)."

#: lib/fb_tools/pdns/record.py:494
msgid "Record {!r} is not in Record list."
msgstr "Der Record {!r} ist nicht in der Recordliste."

#: lib/fb_tools/pdns/record.py:659 lib/fb_tools/pdns/zone.py:131
msgid "Given data {!r} is not a dict object."
msgstr "Die übergebenen Daten {!r} sind kein dict-Objekt."

#: lib/fb_tools/pdns/record.py:662
msgid "Creating {} object from data:"
msgstr "Erstelle ein {}-Objekt aus Daten:"

#: lib/fb_tools/pdns/record.py:798
msgid "Cannot create {o} from record set:"
msgstr "Kann kein {o} aus RecordSet erstellen:"

#: lib/fb_tools/pdns/record.py:803
msgid "RecordSet has no records:"
msgstr "der RecordSet hat keine Records:"

#: lib/fb_tools/pdns/record.py:811
msgid "Got SOA:"
msgstr "Erhielt SOA:"

#: lib/fb_tools/pdns/record.py:876
msgid "RecordSet {n!r} ({n}) is not in RecordSet list."
msgstr "Der RecordSet {n!r} ({n}) befindet sich nicht in der RecordSet-Liste."

#: lib/fb_tools/pdns/server.py:75
msgid "Setting simulate of all subsequent objects to {!r} ..."
msgstr "Setze Simulationsmodus für alle untergeordneten Objekte auf {!r} …"

#: lib/fb_tools/pdns/server.py:106
msgid "Could not found server info."
msgstr "Konnte Serverinformationen nicht finden."

#: lib/fb_tools/pdns/server.py:109 lib/fb_tools/pdns/server.py:126
msgid "Got a response:"
msgstr "Erhaltene Antwort:"

#: lib/fb_tools/pdns/server.py:113
msgid "PowerDNS server version {!r}."
msgstr "Version des PowerDNS-Servers {!r}."

#: lib/fb_tools/pdns/server.py:115
msgid "Did not found version info in server info:"
msgstr "Keine Version in den Serverinformationen gefunden:"

#: lib/fb_tools/pdns/server.py:121
msgid "Trying to get all zones from PDNS API ..."
msgstr "Versuche alle Zonen der PDNS-API zu erhalten …"

#: lib/fb_tools/pdns/server.py:141
msgid "Found a zone."
msgid_plural "Found {n} zones."
msgstr[0] "Fand eine Zone."
msgstr[1] "Fand {n} Zonen."

#: lib/fb_tools/pdns/server.py:146 lib/fb_tools/pdns/server.py:148
msgid "Zones:"
msgstr "Zonen:"

#: lib/fb_tools/pdns/server.py:163
msgid "Searching an appropriate zone for item {i!r} - FQDN {f!r} ..."
msgstr "Suche eine entsprechende Zone für Eintrag {i!r} - FQDN {f!r} …"

#: lib/fb_tools/pdns/server.py:169 lib/fb_tools/pdns/server.py:200
msgid "Search pattern: {}"
msgstr "Suchmuster: {}"

#: lib/fb_tools/pdns/server.py:176 lib/fb_tools/pdns/server.py:208
msgid "Search pattern Unicode: {}"
msgstr "Suchmuster Unicode: {}"

#: lib/fb_tools/pdns/server.py:193
msgid "Searching all appropriate zones for item {i!r} - FQDN {f!r} ..."
msgstr "Suche alle entsprechenden Zonen für Eintrag {i!r} - FQDN {f!r} …"

#: lib/fb_tools/pdns/zone.py:55
msgid "No Resource Record Sets found to remove from zone {!r}."
msgstr "Keine ResourceRecord-Sets gefunden, die man von der Zone {!r} löschen kann."

#: lib/fb_tools/pdns/zone.py:187
msgid "Params initialisation:"
msgstr "Initialisierung der Parameter:"

#: lib/fb_tools/pdns/zone.py:459
msgid "Invalid source tuples for detecting IPv4-network: {!r}."
msgstr "Ungültige Source-Tupel zur Ermittlung des IPv4-Netzwerkes: {!r}."

#: lib/fb_tools/pdns/zone.py:526
msgid "Copying current {}-object into a new one."
msgstr "Kopiere das aktuelle {}-Objekt in ein neues."

#: lib/fb_tools/pdns/zone.py:548
msgid "Cannot update zone {!r}, no API URL defined."
msgstr "Konnte Zone {!r} nicht aktualisieren, keine API-URL definiert."

#: lib/fb_tools/pdns/zone.py:551
msgid "Updating data of zone {n!r} from API path {u!r} ..."
msgstr "Aktualisiere Daten der Zone {n!r} vom API-Pfad {u!r} …"

#: lib/fb_tools/pdns/zone.py:638
msgid "Patching zone {!r} ..."
msgstr "Patche Zone {!r} …"

#: lib/fb_tools/pdns/zone.py:655
msgid "Did not get SOA for zone {!r}."
msgstr "Keinen SOA für Zone {!r} erhalten."

#: lib/fb_tools/pdns/zone.py:662
msgid "New SOA must be of type {e}, given {t}: {s!r}"
msgstr "Neuer SOA muss vom Typ {e}, übergeben {t}: {s!r}"

#: lib/fb_tools/pdns/zone.py:673
msgid "Got no SOA for zone {!r}."
msgstr "Keinen SOA für Zone {!r} erhalten."

#: lib/fb_tools/pdns/zone.py:709
msgid "Setting new SOA {s!r} for zone {z!r}, TTL {t} ..."
msgstr "Setze neuen SOA {s!r} für Zone {z!r}, TTL {t} …"

#: lib/fb_tools/pdns/zone.py:723
msgid "Increasing serial of zone {z!r} from {o} => {n}."
msgstr "Erhöhe die Seriennummer der Zone {z!r} von {o} => {n}."

#: lib/fb_tools/pdns/zone.py:731
msgid "Parameter address {a!r} is not an IPv4Address or IPv6Address object, but a {c} object instead."
msgstr ""
"Der Parameter Adresse {a!r} ist kein IPv4Address- oder IPv6Address-Objekt, ist statt dessen ein "
"{c}-Objekt."

#: lib/fb_tools/pdns/zone.py:739
msgid "Trying to create {t}-record {f!r} => {a!r}."
msgstr "Versuche einen {t}-Eintrag {f!r} => {a!r} zu erstellen."

#: lib/fb_tools/pdns/zone.py:746 lib/fb_tools/pdns/zone.py:804 lib/fb_tools/pdns/zone.py:880
msgid "Current zone:"
msgstr "Aktuelle Zone:"

#: lib/fb_tools/pdns/zone.py:790 lib/fb_tools/pdns/zone.py:849
msgid "Creating {t}-record {f!r} => {a!r}."
msgstr "Erstelle {t}-Record {f!r} => {a!r}."

#: lib/fb_tools/pdns/zone.py:891
msgid "DNS {t!r}-record {n!r} is already deleted."
msgstr "Der DNS-{t}-Record wurde bereits gelöscht."

#: lib/fb_tools/pdns/zone.py:901
msgid "Removing one resource record set from zone {z!r}."
msgid_plural "Removing {c} resource record sets from zone {z!r}."
msgstr[0] "Lösche einen ResourceRecord-Set von Zone {z!r}."
msgstr[1] "Lösche {c} ResourceRecord-Sets von Zone {z!r}."

#: lib/fb_tools/pdns/zone.py:907
msgid "Resorce record sets:"
msgstr "Ressourcen-RecordSet:"

#: lib/fb_tools/pdns/zone.py:910
msgid "Done."
msgstr "Erledigt."

#: lib/fb_tools/pdns/zone.py:917
msgid "Notifying slave servers of zone {!r} ..."
msgstr "Benachrichtige Slave-Server der Zone {!r} …"

#: lib/fb_tools/pdns/zone.py:933 lib/fb_tools/vsphere/ds.py:338 lib/fb_tools/vsphere/ds_cluster.py:180
#: lib/fb_tools/vsphere/network.py:201
msgid "Invalid item type {{!r}} to set, only {} allowed."
msgstr "Ungültiger Typ {{!r}} des Eintrags beim Setzen, nur {} ist erlaubt."

#: lib/fb_tools/pdns/zone.py:935
msgid "The key {k!r} must be equal to the zone name {n!r}."
msgstr "Der Key {k!r} muss gleich dem Zonennamen {n!r} sein."

#: lib/fb_tools/pdns/zone.py:936 lib/fb_tools/vsphere/ds.py:341 lib/fb_tools/vsphere/ds_cluster.py:183
#: lib/fb_tools/vsphere/network.py:204
msgid "None type as key is not allowed."
msgstr "Ein None-Typ ist als Key nicht erlaubt."

#: lib/fb_tools/pdns/zone.py:937 lib/fb_tools/vsphere/ds.py:342 lib/fb_tools/vsphere/ds_cluster.py:184
#: lib/fb_tools/vsphere/network.py:205
msgid "Empty key {!r} is not allowed."
msgstr "Ein leerer Key {!r} ist als Key nicht erlaubt."

#: lib/fb_tools/pdns/zone.py:938
msgid "Object {o!r} is not a {e} object."
msgstr "Das Objekt {o!r} ist kein {e}-Objekt."

#: lib/fb_tools/pdns_bulk_rm_app.py:68
msgid ""
"Removes the given addresses (A-, AAAA- or CNAME-Records) completety from PowerDNS. If there are "
"multiple entries to a DNS-Name, all appropriate records are removed. Additionally all "
"appropriate reverse entries (PTR-records) were also removed, if they are pointing back to the "
"given A- or AAAA-record."
msgstr ""
"Löscht die übergebenen Adressen (A-, AAAA- oder CNAME-Records) vollständig vom PowerDNS. Wenn es"
" mehrere Einträge zu diesem DNS-Namen gibt, werden alle entsprechenden Record gelöscht. "
"Zusätzlich werden die entsprechenden Reverse-Einträge ((PTR-Records) gelöscht, wenn sie auf die "
"entsprechend übergebenen A- bzw. AAAA-Records zeigen."

#: lib/fb_tools/pdns_bulk_rm_app.py:164
msgid "No addresses to remove given."
msgstr "Es wurden keine Adressen zum Löschen übergeben."

#: lib/fb_tools/pdns_bulk_rm_app.py:194
msgid "PowerDNS options"
msgstr "PowerDNS-Optionen"

#: lib/fb_tools/pdns_bulk_rm_app.py:197
msgid "HOST"
msgstr "HOST"

#: lib/fb_tools/pdns_bulk_rm_app.py:198
msgid "Address or hostname of the PowerDNS server providing the API (Default: {!r})."
msgstr "Adresse oder Hostname des PowerDNS-Servers, der die API zur Verfügung stellt (Vorgabe: {!r})."

#: lib/fb_tools/pdns_bulk_rm_app.py:204
msgid "PORT"
msgstr "PORT"

#: lib/fb_tools/pdns_bulk_rm_app.py:205
msgid "Port on PowerDNS server for API on (Default: {})."
msgstr "Port des PowerDNS-Servers für die API (Vorgabe: {})."

#: lib/fb_tools/pdns_bulk_rm_app.py:209
msgid "The API key for accessing the PowerDNS API."
msgstr "Der API-Key für den Zugriff auf die PowerDNS-API."

#: lib/fb_tools/pdns_bulk_rm_app.py:214
msgid "Use HTTPS to access the PowerDNS API (Default: {})."
msgstr "Verwendung von HTTPS für den Zugriff auf die PowerDNS-API (Vorgabe: {})."

#: lib/fb_tools/pdns_bulk_rm_app.py:220
msgid "The global prefix for all paths for accessing the PowerDNS API (Default: {!r})."
msgstr "Ther globale Prefix für alle Pfade ür den Zugriff auf die PowerDNS-API (Vorgabe: {!r})."

#: lib/fb_tools/pdns_bulk_rm_app.py:230
msgid ""
"Don't remove reverse DNS entries (PTR records) to the given addresses. (Default: False - reverse"
" entries will be removed)."
msgstr ""
"Kein Löschen der Reverse-DNS-Einträge (PTR-Records) zu den übergebenen Adressen (Vorgabe: Falsch"
" -  Reverse-Einträge werden gelöscht)."

#: lib/fb_tools/pdns_bulk_rm_app.py:237
msgid ""
"File containing the addresses to remove. The addresses must be whitespace separeted, lines may "
"be commented out by prepending them with a hash sign '#'. This option is mutually exclusive with"
" giving the addresses as command line arguments."
msgstr ""
"Eine Datei, die die zu löschenden Adressen enthält. iDie Adressen müssen Leerzeichen-getrennt "
"sein. Zeilen können auskommentiert werden, indem man ein Hash-Zeichen '#' an den Anfang der "
"Zeile stellt. Diese Option schlißt sich gegenseitig mit der Übergabe der Adressen als "
"Kommandozeilenargumente aus."

#: lib/fb_tools/pdns_bulk_rm_app.py:245
msgid "ADDRESS"
msgstr "ADRESSE"

#: lib/fb_tools/pdns_bulk_rm_app.py:246
msgid "Addresses to remove. This option is mutually exclusive with the {!r} option."
msgstr ""
"Adressen, die gelöscht werden sollen. Diese Option schlißt sich gegenseitig mit der {!r}-Option "
"aus."

#: lib/fb_tools/pdns_bulk_rm_app.py:265
msgid "The option {!r} is mutually exclusive with giving the addresses as command line arguments."
msgstr ""
"Die Option {!r} schließt sich gegenseitig mit der Übergabe der Adressen als "
"Kommandozeilenargumente aus."

#: lib/fb_tools/pdns_bulk_rm_app.py:283
msgid "No read access to file {!r}."
msgstr "Kein Lesezugriff für die Datei {!r}."

#: lib/fb_tools/pdns_bulk_rm_app.py:332
msgid "No addresses to remove found in {!r}."
msgstr "Keine Adressen zum Löschen in der Datei {!r} gefunden."

#: lib/fb_tools/pdns_bulk_rm_app.py:371
msgid "Simulation mode - nothing will be removed in real."
msgstr "Simulations-Modus - es wird in Wirklichkeit nichts gelöscht."

#: lib/fb_tools/pdns_bulk_rm_app.py:399
msgid "Address {!r} could not interpreted as a FQDN."
msgstr "Die Adresse {!r} konnte nicht als FQDN interpretiert werden."

#: lib/fb_tools/pdns_bulk_rm_app.py:419
msgid "Did not found an appropriate zone for address {!r}."
msgstr "Keine passende Zone für die Adresse {!r} gefunden."

#: lib/fb_tools/pdns_bulk_rm_app.py:495
msgid "PTR {p!r} does not pointing to expected {e!r}, but to {c!r} instead, ignoring for deletion."
msgstr ""
"Der PTR {p!r} zeigt nicht auf die erwartete Adresse {e!r}, sondern statt dessen auf {c!r}. Er "
"wird zum Löschen ignoriert."

#: lib/fb_tools/pdns_bulk_rm_app.py:521
msgid "Did not found any addresses with an appropriate zone in PowerDNS."
msgstr "Keine Adressen mit entsprechenden Zonen in PowerDNS gefunden."

#: lib/fb_tools/pdns_bulk_rm_app.py:539
msgid "The following addresses (FQDNs) are not found:"
msgstr "Die folgenden Adressen (FQDNs) wurden nicht gefunden:"

#: lib/fb_tools/pdns_bulk_rm_app.py:577
msgid "IP address {!r} seems not to be a valid IP address."
msgstr "IDie IP-Adresse {!r} scheint keine gültige IP-Adresse zu sein."

#: lib/fb_tools/pdns_bulk_rm_app.py:597
msgid "All DNS records to remove"
msgstr "Alle zu löschenden DNS-Records"

#: lib/fb_tools/pdns_bulk_rm_app.py:603
msgid "Disabled."
msgstr "Deaktiviert"

#: lib/fb_tools/pdns_bulk_rm_app.py:605
msgid "Name"
msgstr "Name"

#: lib/fb_tools/pdns_bulk_rm_app.py:606
msgid "Zone"
msgstr "Zone"

#: lib/fb_tools/pdns_bulk_rm_app.py:607
msgid "Type"
msgstr "Typ"

#: lib/fb_tools/pdns_bulk_rm_app.py:608
msgid "Record"
msgstr "Record"

#: lib/fb_tools/pdns_bulk_rm_app.py:664
msgid "Total one DNS record to remove."
msgid_plural "Total {} DNS records to remove."
msgstr[0] "Insgesamt ein DNS-Record, der gelöscht wird."
msgstr[1] "Insgesamt {} DNS-Records, die gelöscht werden."

#: lib/fb_tools/syslog_handler.py:76
msgid "File doesn't exists"
msgstr "Die Datei existiert nicht"

#: lib/fb_tools/syslog_handler.py:80
msgid "File is not a UNIX socket file"
msgstr "Die Datei ist keine UNIX-Socket-Datei"

#: lib/fb_tools/syslog_handler.py:83
msgid "No write access to socket"
msgstr "Kein Schreibzugriff auf den Socket"

#: lib/fb_tools/unix_handler.py:220 lib/fb_tools/unix_handler.py:226
msgid "Invalid value {!r} for facility."
msgstr "Ungültiger Wert für Fazilität."

#: lib/fb_tools/vsphere/__init__.py:104
msgid "Invalid port number {!r} for the VSphere server, must be 0 < PORT <= 65536."
msgstr "Ungültige Portnummer {!r} für den vSphere-Server, muss 0 < PORT <= 65536 sein."

#: lib/fb_tools/vsphere/__init__.py:196
msgid "Connecting to vSphere host {h}:{p} as {u!r} ..."
msgstr "Verbinde mich mit vSphere-Host {h}:{p} als {u!r} …"

#: lib/fb_tools/vsphere/__init__.py:214
msgid "Disconnecting from vSphere host {}."
msgstr "Trenne Verbindung zu vSphere-Host {}."

#: lib/fb_tools/vsphere/cluster.py:282 lib/fb_tools/vsphere/dc.py:154 lib/fb_tools/vsphere/ds.py:233
#: lib/fb_tools/vsphere/ds_cluster.py:123 lib/fb_tools/vsphere/network.py:144
msgid "Creating {} object from:"
msgstr "Erstelle {}-Objekt aus:"

#: lib/fb_tools/vsphere/cluster.py:290
msgid "Cluster {c!r} has network {n!r}."
msgstr "Der Cluster {c!r} hat das Netzwerk {n!r}."

#: lib/fb_tools/vsphere/cluster.py:297
msgid "Cluster {c!r} has datastore {d!r}."
msgstr "Der Cluster {c!r} hat den Datastore {d!r}."

#: lib/fb_tools/vsphere/ds.py:340
msgid "The key {k!r} must be equal to the datastore name {n!r}."
msgstr "Der Schlüssel {k!r} muss gleich dem Namen des Datastores {n!r} sein."

#: lib/fb_tools/vsphere/ds.py:343 lib/fb_tools/vsphere/ds_cluster.py:185
#: lib/fb_tools/vsphere/network.py:206
msgid "Object {{!r}} is not a {} object."
msgstr "Das Objekt {{!r}} ist kein {}-Objekt."

#: lib/fb_tools/vsphere/ds.py:575
msgid "Could not handle datastore type {!r}."
msgstr "Kann den Datastore-Typ {!r} nicht behandeln."

#: lib/fb_tools/vsphere/ds.py:582
msgid "Could not found a datastore for {c:0.1f} GiB of type {t!r}."
msgstr "Konnte keinen Datastore für {c:0.1f} GiB vom Typ {t!r} finden."

#: lib/fb_tools/vsphere/ds.py:589
msgid "Searching datastore for {c:0.1f} GiB of type {t!r}."
msgstr "Suche Datastore für {c:0.1f} GiB vom Typ {t!r}."

#: lib/fb_tools/vsphere/ds_cluster.py:182
msgid "The key {k!r} must be equal to the datastore cluster name {n!r}."
msgstr "Der Key {k!r} muss gleich dem Namen des Datastore-Clusters {n!r} sein."

#: lib/fb_tools/vsphere/errors.py:34
msgid "No VSphere datastores found."
msgstr "Keine VSphere-Datastores gefunden."

#: lib/fb_tools/vsphere/errors.py:64
msgid "Invalid name {n!r} for a {o} VSphere object."
msgstr "Ungültiger Name für ein {o}-VSphere-Objekt."

#: lib/fb_tools/vsphere/errors.py:67
msgid "Invalid name {!r} for a VSphere object."
msgstr "Ungültiger Name für ein VSphere-Objekt."

#: lib/fb_tools/vsphere/errors.py:85
msgid "The VSphere datacenter {!r} is not existing."
msgstr "Das VSphere-Datacenter existiert nicht."

#: lib/fb_tools/vsphere/errors.py:101
msgid "The VSphere Virtual machine {!r} was not found."
msgstr "Die VSphere-VM {!r} wurde nicht gefunden."

#: lib/fb_tools/vsphere/errors.py:121
msgid "No SAN based datastore found with at least {m:0.0f} MiB == {g:0.1f} GiB available space found."
msgstr ""
"Kein SAN-basierter Datastore mit wenigstens {m:0.0f} MiB == {g:0.1f} GiB verfügbarem Platz "
"gefunden."

#: lib/fb_tools/vsphere/errors.py:181
msgid "Timeout on creating VM {vm!r} after {to:0.1f} seconds."
msgstr "Timeout bei der Erstellung der VM {vm!r} nach {to:0.1f} Sekunden."

#: lib/fb_tools/vsphere/iface.py:122
msgid "Invalid MAC address {!r} for interface given."
msgstr "Ungültige MAC-Adresse {!r} für das Interface übergeben."

#: lib/fb_tools/vsphere/network.py:65
msgid "Trying to get IPv4 network {n!r} -> {i!r}."
msgstr "Versuche das IPv4-Netzwerk {n!r} -> {i!r} zu ermitteln."

#: lib/fb_tools/vsphere/network.py:72
msgid "Could not get IP network from network name {!r}."
msgstr "Konnte das IP-Netzwerk nicht aus dem Netzwerknamen {!r} ermitteln."

#: lib/fb_tools/vsphere/network.py:75
msgid "Network {!r} has no IP network assigned."
msgstr "Dem Netzwerk {!r} wurde kein IP-Netzwerk zugewiesen."

#: lib/fb_tools/vsphere/network.py:81
msgid "Initialized network object:"
msgstr "Initialisiertes Netzwerk-Objekt:"

#: lib/fb_tools/vsphere/network.py:203
msgid "The key {k!r} must be equal to the network name {n!r}."
msgstr "Der Key {k!r} muss gleich dem Netzwerknamen {n!r} sein."

#: lib/fb_tools/vsphere/network.py:451
msgid "Searching VSphere network for address {} ..."
msgstr "Suche VSphere-Netzwerk für die Adresse {} …"

#: lib/fb_tools/vsphere/network.py:457
msgid "Found network {n!r} for IP {i}."
msgstr "Habe Netzwerk {n!r} für die IP {i} gefunden."

#: lib/fb_tools/vsphere/network.py:461
msgid "Could not find VSphere network for IP {}."
msgstr "Konnte kein VSphere-Netzwerk für die IP {i} finden."

#: lib/fb_tools/vsphere/network.py:464
msgid "Could not find VSphere network for IP addresses {}."
msgstr "Konnte kein VSphere-Netzwerk für die IP-Adresse {} ermitteln."

#: lib/fb_tools/vsphere/object.py:76
msgid "The type of a {} may not be None."
msgstr "Der Typ von einem {} darf nicht None sein."

#: lib/fb_tools/vsphere/object.py:81 lib/fb_tools/vsphere/object.py:104
#: lib/fb_tools/vsphere/object.py:127
msgid "Invalid {c}.{p} {v!r}."
msgstr "Ungültiger {c}.{p} {v!r}."

#: lib/fb_tools/vsphere/object.py:143
msgid "The name prefix of a {} may not be None."
msgstr "Der Namenspräfix eines {} darf nicht None sein."

#: lib/fb_tools/vsphere/object.py:147
msgid "Invalid name prefix {p!r} for a {o}."
msgstr "Ungültiger Namenspräfix {p!r} für ein {o}."

#: lib/fb_tools/vsphere/server.py:110
msgid "Trying to get some 'about' information from VSphere."
msgstr "Versuche, ein paar 'about'-Informationen von VSphere zu ermitteln."

#: lib/fb_tools/vsphere/server.py:166
msgid "VSphere server version: {!r}"
msgstr "VSphere-Server-Version: {!r}"

#: lib/fb_tools/vsphere/server.py:168
msgid "Found VSphere about-information:"
msgstr "Gefundene VSphere-'about'-Informationen:"

#: lib/fb_tools/vsphere/server.py:173
msgid "Trying to get datacenter from VSphere ..."
msgstr "Versuche, Datacenter von VSphere zu ermitteln …"

#: lib/fb_tools/vsphere/server.py:187
msgid "Found VSphere datacenter {!r}."
msgstr "VSphere-Datacenter {!r} gefunden."

#: lib/fb_tools/vsphere/server.py:189
msgid "Info about datacenter:"
msgstr "Informationen über Datacenter:"

#: lib/fb_tools/vsphere/server.py:200
msgid "Trying to get all clusters from VSphere ..."
msgstr "Versuche alle Cluster von VSphere zu ermitteln …"

#: lib/fb_tools/vsphere/server.py:224 lib/fb_tools/vsphere/server.py:229
msgid "Found clusters:"
msgstr "Gefundene Cluster:"

#: lib/fb_tools/vsphere/server.py:245
msgid ""
"Found cluster {cl!r}, {h} hosts, {cpu} CPUs, {thr} threads, {mem:0.1f} GiB Memory, {net} "
"networks and {ds} datastores."
msgstr ""
"Gefundenes Cluster {cl!r}, {h} Hosts, {cpu} CPUs, {thr} Threads, {mem:0.1f} GiB Hauptspeicher, "
"{net} Netzwerke und {ds} Datastores."

#: lib/fb_tools/vsphere/server.py:267
msgid "Trying to get all datastores from VSphere ..."
msgstr "Versuche alle Datastores von VSphere zu ermitteln …"

#: lib/fb_tools/vsphere/server.py:290 lib/fb_tools/vsphere/server.py:292
msgid "Found datastores:"
msgstr "Gefundene Datastores:"

#: lib/fb_tools/vsphere/server.py:300
msgid "Datastore mappings:"
msgstr "Datastore-Mappings:"

#: lib/fb_tools/vsphere/server.py:315
msgid "Datastore {!r} seems to be local."
msgstr "Das Datastore {!r} scheint lokal zu sein."

#: lib/fb_tools/vsphere/server.py:321
msgid "Found datastore {ds!r} of type {t!r}, capacity {c:0.1f} GByte."
msgstr "Habe Datastore {ds!r} vom Typ {t!r} mit einer Kapazität von {c:0.1f} GByte gefunden."

#: lib/fb_tools/vsphere/server.py:330
msgid "Trying to get all datastore clusters from VSphere ..."
msgstr "Versuche alle Datastore-Cluster von VSphere zu ermitteln …"

#: lib/fb_tools/vsphere/server.py:354 lib/fb_tools/vsphere/server.py:357
msgid "Found datastore clusters:"
msgstr "Gefundene Datastore-Cluster:"

#: lib/fb_tools/vsphere/server.py:359
msgid "No VSphere datastore clusters found."
msgstr "Keine VSphere-Datastore-Cluster gefunden."

#: lib/fb_tools/vsphere/server.py:365
msgid "Datastore cluster mappings:"
msgstr "Datastore-Cluster-Mappings:"

#: lib/fb_tools/vsphere/server.py:371 lib/fb_tools/vsphere/server.py:431
#: lib/fb_tools/vsphere/server.py:491 lib/fb_tools/vsphere/server.py:657
msgid "Found a {} child."
msgstr "Habe ein {}-Kindobjekt gefunden."

#: lib/fb_tools/vsphere/server.py:389
msgid "Trying to get all networks from VSphere ..."
msgstr "Versuche alle Netzwerke von VSphere zu ermitteln …"

#: lib/fb_tools/vsphere/server.py:411
msgid "Found one VSphere network."
msgid_plural "Found {n} VSphere networks."
msgstr[0] "Habe ein VSphere-Netzwerk gefunden."
msgstr[1] "Habe {n} VSphere-Netzwerke gefunden."

#: lib/fb_tools/vsphere/server.py:415 lib/fb_tools/vsphere/server.py:417
msgid "Found VSphere networks:"
msgstr "Gefundene VSphere-Netzwerke:"

#: lib/fb_tools/vsphere/server.py:419
msgid "No VSphere networks found."
msgstr "Keine VSphere-Netzwerke gefunden."

#: lib/fb_tools/vsphere/server.py:425
msgid "Network mappings:"
msgstr "Netzwerk-Mappings:"

#: lib/fb_tools/vsphere/server.py:449
msgid "Trying to get VM {!r} from VSphere ..."
msgstr "Versuche die VM {!r} von VSphere zu ermitteln …"

#: lib/fb_tools/vsphere/server.py:465 lib/fb_tools/vsphere/server.py:489
#: lib/fb_tools/vsphere/server.py:634 lib/fb_tools/vsphere/server.py:655
msgid "Searching in path {!r} ..."
msgstr "Suche in Pfad {!r} …"

#: lib/fb_tools/vsphere/server.py:475
msgid "VSphere VM {!r} not found."
msgstr "Die VSphere-VM {!r} wurde nicht gefunden."

#: lib/fb_tools/vsphere/server.py:513
msgid "Found VM {r} summary:"
msgstr "Gefundene Zusammenfassung der VM {!r}:"

#: lib/fb_tools/vsphere/server.py:515
msgid "Found VM {!r} config:"
msgstr "Gefundene Konfiguration der VM {!r}:"

#: lib/fb_tools/vsphere/server.py:614
msgid "Parameter {p!r} => {r!r} seems not to be a regex object."
msgstr "Der Parameter {p!r} => {r!r} scheint kein Regex-Objekt zu sein."

#: lib/fb_tools/vsphere/server.py:618
msgid "Trying to get list of VMs with name pattern {!r} ..."
msgstr "Versuche eine Liste der VMs mit dem Namensmuster {!r} zu finden …"

#: lib/fb_tools/vsphere/server.py:644
msgid "Found one VM with pattern {p!r}."
msgid_plural "Found {no} VMs with pattern {p!r}."
msgstr[0] "Eine VM mit dem Namensmuster {p!r} gefunden."
msgstr[1] "Habe {no} VMs mit dem Namensmuster {p!r} gefunden."

#: lib/fb_tools/vsphere/server.py:682
msgid "Checking VM {!r} ..."
msgstr "Überprüfe VM {!r} …"

#: lib/fb_tools/vsphere/server.py:685
msgid "Checking VM {!r} for being a template ..."
msgstr "Überprüfe VM {!r}, ob sie eine Vorlage ist …"

#: lib/fb_tools/vsphere/server.py:687
msgid "Checking VM {!r} for being not a template ..."
msgstr "Überprüfe VM {!r}, ob sie keine Vorlage ist …"

#: lib/fb_tools/vsphere/server.py:695
msgid "Checking VM {!r} for pattern."
msgstr "Überprüfe VM {!r} nach Muster."

#: lib/fb_tools/vsphere/server.py:698
msgid "Found VM {!r}."
msgstr "Habe VM {!r} gefunden."

#: lib/fb_tools/vsphere/server.py:725
msgid "VM {!r} is already powered on."
msgstr "Die VM {!r} ist bereits eingeschaltet."

#: lib/fb_tools/vsphere/server.py:728
msgid "Powering on VM {!r} ..."
msgstr "Schalte VM {!r} ein …"

#: lib/fb_tools/vsphere/server.py:732
msgid "VM {!r} successful powered on."
msgstr "Die VM {!r} wurde erfolgreich eingeschaltet."

#: lib/fb_tools/vsphere/server.py:756
msgid "VM {!r} is already powered off."
msgstr "Die VM {!r} ist bereits ausgeschaltet."

#: lib/fb_tools/vsphere/server.py:759
msgid "Powering off VM {!r} ..."
msgstr "Schalte VM {!r} aus …"

#: lib/fb_tools/vsphere/server.py:763
msgid "VM {!r} successful powered off."
msgstr "Die VM {!r} wurde erfolgreich ausgeschaltet."

#: lib/fb_tools/vsphere/server.py:771
msgid "Ensuring existence of VSphere VM folders:"
msgstr "Stelle Existenz von VSphere VM-Ordnern sicher:"

#: lib/fb_tools/vsphere/server.py:788
msgid "Trying to get VM folder object for path {!r}."
msgstr "Versuche das VM-Ordnerobjekt für den Pfad {!r} zu erhalten."

#: lib/fb_tools/vsphere/server.py:813
msgid "Checking single VM folder {i}: {f!r}."
msgstr "Überprüfe einfachen VM-Ordner {i}: {f!r}."

#: lib/fb_tools/vsphere/server.py:824
msgid "Found VM folder {n}, parent: {p}"
msgstr "Habe VM-Ordner {n} gefunden, Eltern-Ordner: {p}"

#: lib/fb_tools/vsphere/server.py:842
msgid "Ensuring existence of VSphere VM folder {!r}."
msgstr "Stelle Existenz des VSphere-VM-Ordners {!r} sicher."

#: lib/fb_tools/vsphere/server.py:867
msgid "VM Folder {!r} already exists."
msgstr "Der VM-Ordner {!r} existiert bereits."

#: lib/fb_tools/vsphere/server.py:869
msgid "Creating VM folder {!r} ..."
msgstr "Erstelle VM-Ordner {!r} …"

#: lib/fb_tools/vsphere/server.py:871
msgid "Simulation mode, don't creating it."
msgstr "Simulationsmodus, erstelle ihn nicht."

#: lib/fb_tools/vsphere/server.py:887
msgid "Waiting for tasks to finish ..."
msgstr "Warte auf Beendigung der Tasks …"

#: lib/fb_tools/vsphere/server.py:899
msgid "Waiting at most {m} seconds for tasks {t} to finish ..."
msgstr "Warte maximal {m} Sekunden auf die Beendigung der Tasks {t} …"

#: lib/fb_tools/vsphere/server.py:902
msgid "Waiting for tasks {} to finish ..."
msgstr "Warte auf Beendigung der Tasks {} …"

#: lib/fb_tools/vsphere/server.py:922
msgid "Waiting ..."
msgstr "Warten …"

#: lib/fb_tools/vsphere/server.py:956
msgid "Creating VM {!r} ..."
msgstr "Erstelle VM {!r} …"

#: lib/fb_tools/vsphere/server.py:959
msgid "Simulation mode - VM {!r} will not be created."
msgstr "Simulations-Modus - die VM {!r} wird nicht erstellt."

#: lib/fb_tools/vsphere/server.py:977
msgid "Generating create spec for VM {!r} ..."
msgstr "Generiere Erstellungs-Spec für VM {!r} …"

#: lib/fb_tools/vsphere/server.py:985
msgid "Datastore path: {!r}"
msgstr "Datastore-Pfad: {!r}"

#: lib/fb_tools/vsphere/server.py:988
msgid "VM path name: {!r}"
msgstr "VM-Pfadname: {!r}"

#: lib/fb_tools/vsphere/server.py:1038
msgid "Generated VM config:"
msgstr "Generierte VM-Konfiguration:"

#: lib/fb_tools/vsphere/server.py:1047
msgid "Given disksize {!r} must be greater than zero."
msgstr "Die gegebene Plattengröße muss größer als Null sein."

#: lib/fb_tools/vsphere/server.py:1060
msgid "There may be created at most 6 disks, but {} were given."
msgstr "Es können höchjstens 6 Platten erstellt werden, aber {} wurden übergeben."

#: lib/fb_tools/vsphere/server.py:1074
msgid "Generating spec for SCSI controller and one disk: {d}"
msgid_plural "Generating spec for SCSI controller and {n} disks: {d}"
msgstr[0] "Generiere Spec für SCSI-Controller und eine Disk: {d}"
msgstr[1] "Generiere Spec für SCSI-Controller und {n} Disks: {d}"

#: lib/fb_tools/vsphere/server.py:1077
msgid "Generating spec for SCSI controller without disks."
msgstr "Generiere Spec für SCSI-Controller ohne Disks."

#: lib/fb_tools/vsphere/server.py:1102
msgid "Adding spec for disk {n!r} with {gb} GiB => {kb} KiByte."
msgstr "Füge Spec für Disk {n!r} mit {gb} GiB => {kb} KiByte hinzu."

#: lib/fb_tools/vsphere/server.py:1137
msgid "Invalid Interface description {!r} given."
msgstr "Ungültige Interface-Beschreibung {!r} übergeben."

#: lib/fb_tools/vsphere/server.py:1148
msgid "Defined interface:"
msgstr "Definierte Netzwerkschnittstelle:"

#: lib/fb_tools/vsphere/server.py:1152
msgid "Adding spec for network interface {d!r} (Network {n!r}, MAC: {m!r}, summary: {s!r})."
msgstr ""
"Füge Spec für Netzwerkschnittstelle {d!r} (Netzwerk {n!r}, MAC: {m!r}, Zusammenfassung: {s!r}) "
"hinzu."

#: lib/fb_tools/vsphere/server.py:1204
msgid "Purging VM {!r} ..."
msgstr "Lösche VM {!r} vollständig …"

#: lib/fb_tools/vsphere/server.py:1208
msgid "VM {!r} successful removed."
msgstr "VM {!r} wurde erfolgreich gelöscht."

#: lib/fb_tools/vsphere/server.py:1239
msgid "Did not found virtual ethernet device No. {no} (found {count} devices)."
msgstr "Virtuelles Ethernet-Device Nummer {no} wurde nicht gefunden (es wurden {count} Devices gefunden)."

#: lib/fb_tools/vsphere/server.py:1258
msgid "Changes of MAC address:"
msgstr "Ändern von MAC-Adressen:"

#: lib/fb_tools/vsphere/server.py:1262
msgid "Successful changed MAC address of VM {v!r} to {m!r}."
msgstr "Erfolgreich MAC-Adresse der VM {v!r} auf {m!r} geändert."

#: lib/fb_tools/xlate.py:54
msgid "Module directory: {!r}"
msgstr "Modul-Verzeichnis: {!r}"

#: lib/fb_tools/xlate.py:55
msgid "Base directory: {!r}"
msgstr "Basis-Verzeichnis: {!r}"

#: lib/fb_tools/xlate.py:56
msgid "Locale directory: {!r}"
msgstr "Verzeichnis für Locales: {!r}"

#: lib/fb_tools/xlate.py:57
msgid "Locale domain: {!r}"
msgstr "Locale-Domäne: {!r}"

#: lib/fb_tools/xlate.py:58
msgid "Found .mo-file: {!r}"
msgstr "Gefundene .mo-Datei: {!r}"

